modules/motions/BackgroundPlaCameraAction.h

```diff
@@ -20,28 +20,30 @@ class BackgroundPlaCameraAction : public CompositeMotion {
    * @param _minArea 最小面積
    * @param _roi 動体検出用の注目領域
    * @param _position 撮影位置（0:正面, 1:右, 2:後ろ, 3:左）
+   * @param _isAbsoluteMode 回頭方法 false:相対角度回頭, true:絶対角度回頭
    * @param _kp 回頭PIDのP値
    * @param _ki 回頭PIDのI値
    * @param _kd 回頭PIDのD値
    */
   BackgroundPlaCameraAction(Robot& _robot, bool _isClockwise, int _preTargetAngle,
                             int _postTargetAngle, int _basePower, double _threshold,
-                            double _minArea, const cv::Rect roi, int _position, double _kp = 0.036,
-                            double _ki = 0.02, double _kd = 0.03);
-
+                            double _minArea, const cv::Rect roi, int _position,
+                            bool _isAbsoluteMode, double _kp = 0.036, double _ki = 0.02,
+                            double _kd = 0.03);
 
  private:
-  bool isClockwise = false;  // 回頭方向
-  int preTargetAngle = 90;   // カメラを風景に向けるための回頭角度
-  int postTargetAngle = 90;  // 黒線復帰のための回頭角度
-  int basePower = 50;        // 回頭基準パワー値
-  double threshold = 30.0;   // 風景検出のしきい値
-  double minArea = 400.0;    // 最小面積
-  int position = 0;          // 撮影位置（0:正面, 1:右, 2:後ろ, 3:左）
-  cv::Rect roi;              // 動体検出用の注目領域
-  double kp = 0.036;         // 回頭PIDのP値
-  double ki = 0.02;          // 回頭PIDのI値
-  double kd = 0.03;          // 回頭PIDのD値
+  bool isClockwise = false;     // 回頭方向
+  int preTargetAngle = 90;      // カメラを風景に向けるための回頭角度
+  int postTargetAngle = 90;     // 黒線復帰のための回頭角度
+  int basePower = 50;           // 回頭基準パワー値
+  double threshold = 30.0;      // 風景検出のしきい値
+  double minArea = 400.0;       // 最小面積
+  int position = 0;             // 撮影位置（0:正面, 1:右, 2:後ろ, 3:左）
+  cv::Rect roi;                 // 動体検出用の注目領域
+  bool isAbsoluteMode = false;  // 回頭方法 false:相対角度回頭, true:絶対角度回頭
```

Comment:
Angleも変数名に入れた方が個人的に分かりやすいかな。
```suggestion
  bool isAbsoluteAngleMode = false;  // 回頭方法 false:相対角度回頭, true:絶対角度回頭
```

Reply:
修正しました